embrace-android-sdk/src/test/java/io/embrace/android/embracesdk/session/SessionHandlerTest.kt

```diff
@@ -70,8 +69,6 @@ import org.junit.Test
 internal class SessionHandlerTest {
```

Comment:
At some point, we should move these tests to the appropriate test classes that are testing those objects now